src/drivers/driver_zephyr.c

```diff
@@ -366,6 +366,12 @@ void wpa_drv_zep_event_proc_scan_done(struct zep_drv_if_ctx *if_ctx,
 			event);
 }
 
+void wpa_drv_zep_event_proc_scan_abort(struct zep_drv_if_ctx *if_ctx)
+{
+    wpa_printf(MSG_DEBUG, "%s: Scan aborted by driver", __func__);
+
+    if_ctx->scan_res2_get_in_prog = false;
```

Comment:
Please follow zephyr coding style in these files. So use tabs (size 8) instead of spaces.
https://docs.zephyrproject.org/latest/contribute/guidelines.html#coding-style

Reply:
thanks, will update